site/content/docs/5.1/forms/quantity-selector.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+title: Quantity selector
+description: Use our custom quantity selector in forms for selecting a number.
+group: forms
+toc: true
+---
+
+## Default
+
+Quantity selector is a form element used to select a number.
+
+**Please note that this component is suitable for incremental and decremental display of small numbers, i.e. numbers with 3-4 digits. We do not recommend using it with larger numbers.**
+
+A default value can be specified with the input `value` attribute.
+
+Value will vary between the values defined for the `min` and `max` attributes (negative value are allowed). `min` and `max` values are customizable.
```

Comment:
Is the `step` attribute also customizable ? if yes, maybe just a word in this line would do the job